specification/servicefabricmanagedclusters/resource-manager/Microsoft.ServiceFabric/preview/2023-07-01-preview/managedcluster.json

```diff
@@ -0,0 +1,2192 @@
+            "schema": {
+              "$ref": "#/definitions/ManagedAzResiliencyStatus"
+            }
+          },
+          "default": {
+            "description": "The detailed error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorModel"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ServiceFabric/managedClusters/{clusterName}/getmaintenancewindowstatus": {
+      "post": {
+        "tags": [
+          "managedMaintenanceWindowStatus"
+        ],
+        "summary": "Action to get Maintenance Window Status of the Service Fabric Managed Clusters.",
+        "description": "Action to get Maintenance Window Status of the Service Fabric Managed Clusters.",
+        "operationId": "managedMaintenanceWindowStatus_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          }
+        ],
+        "x-ms-examples": {
+          "Maintenance Window Status": {
+            "$ref": "./examples/ManagedMaintenanceWindowStatusGet_example.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "The operation completed successfully.",
+            "schema": {
+              "$ref": "#/definitions/ManagedMaintenanceWindowStatus"
+            }
+          },
+          "default": {
+            "description": "The detailed error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorModel"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ServiceFabric/managedClusters/{clusterName}/applymaintenancewindow": {
+      "post": {
+        "tags": [
+          "managedApplyMaintenanceWindow"
+        ],
+        "summary": "Action to Apply Maintenance window on the Service Fabric Managed Clusters, right now. Any pending update will be applied.",
+        "description": "Action to Apply Maintenance window on the Service Fabric Managed Clusters, right now. Any pending update will be applied.",
+        "operationId": "managedApplyMaintenanceWindow_Put",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          }
+        ],
+        "x-ms-examples": {
+          "Maintenance Window Status": {
+            "$ref": "./examples/ManagedApplyMaintenanceWindowPut_example.json"
```

Comment:
>ManagedApplyMaintenanceWindowPut_example [](http://example.com/codeflow?start=32&length=40)

Why is 'PUT' in the name of the example when its actually using a POST verb?

Reply:
it was incorrect. fixed